src/index.tsx

```diff
@@ -97,11 +82,7 @@ function ExcalidrawApp() {
   const { width, height } = dimensions;
   return (
     <TopErrorBoundary>
-      <IsMobileProvider>
-        <InitializeApp>
-          <App width={width} height={height} />
-        </InitializeApp>
-      </IsMobileProvider>
+      <Excalidraw width={width} height={height} />
```

Comment:
> App shouldn't save to local storage by default now as its the choice of host app now so it should be handled here, that was the reason I had added `onBlur` and `onChange` props. [excalidraw/excalidraw-embed@e4eaa56](https://github.com/excalidraw/excalidraw-embed/commit/e4eaa566ffb4288188370afbe97f00c073e5a2fc)

I see. Well, we should do that in later PR. We'll also want to decouple network IO (links, collab) from the compo.

> We will need to pass initialData as well.

Pass where/when?

Reply:
> > App shouldn't save to local storage by default now as its the choice of host app now so it should be handled here, that was the reason I had added `onBlur` and `onChange` props. [excalidraw/excalidraw-embed@e4eaa56](https://github.com/excalidraw/excalidraw-embed/commit/e4eaa566ffb4288188370afbe97f00c073e5a2fc)
> 
> I see. Well, we should do that in later PR. We'll also want to decouple network IO (links, collab) from the compo.

I think it would be better to do now since we shouldn't release any version as it would be breaking change for host app and won't make any sense since storing or not in local storage should be decided by host app
> 
> > We will need to pass initialData as well.
> 
> Pass where/when?

Same as above since we are removing local storage code from app so the initial app state needs to be passed.

Reply:
(discussed and will be done in next PR)